src/masternodes/govvariables/attributes.cpp

```diff
@@ -39,15 +39,23 @@ static ResVal<int32_t> VerifyInt32(const std::string& str) {
     return {int32, Res::Ok()};
 }
 
-static ResVal<CAmount> VerifyPct(const std::string& str) {
+static ResVal<CAmount> VerifyFloat(const std::string& str) {
     CAmount amount = 0;
     if (!ParseFixedPoint(str, 8, &amount) || amount < 0) {
-        return Res::Err("Percentage must be a positive integer or float");
+        return Res::Err("Amount must be a float");
```

Comment:
It still could be an integer.

Reply:
Update to "Amount must be positive value"